charts/core/traefik/templates/_args.tpl

```diff
@@ -145,7 +145,7 @@ args:
   - "--pilot.token={{ .Values.pilot.token }}"
   {{- if .Values.pilot.enableThemePark }}
   - "--experimental.plugins.themePark.modulename=github.com/packruler/traefik-themepark"
-  - "--experimental.plugins.themePark.version=v0.2.1"
+  - "--experimental.plugins.themePark.version=v1.0.0"
```

Comment:
@Ornias1993 I wonder if Renovate will pick this up and update it.
Or this will be a manual thing?

Reply:
It looks like we should be able to setup a regex module for handling this and future traefik plugins

Reply:
The multiline aspect makes me unsure about the regex module for renovate.

Reply:
Come to think about it. This may be feasible if we set the version in a variable definition then split it for applying to arguments. 🤔